client/less/styles.less

```diff
@@ -2,15 +2,24 @@
 @import 'comment';
 @import 'commentForm';
 
-*, body {
-  font-family: 'Open Sans', sans-serif;
-  color: #333;
-}
+@import url(https://fonts.googleapis.com/css?family=Nunito:400,700|Open+Sans:400,700);
```

Comment:
Note that this was happening here: https://github.com/MobilityLabs/project-mercury-blade/blob/master/client/build/index.html#L5

If you import the font here, you should be able to ditch the reference in the index.html

Also, have you tried protocol-less asset urls?  I am curious if you could change this to:
```
@import url(//fonts.googleapis.com/css?family=Nunito:400,700|Open+Sans:400,700);
```
I have not tried this in less, but I bet it would work. It is how I do all of my resources, it means that I don't worry about which protocol is used nor the insecure page warning that comes with an accidental http image in an https page. :smile_cat: